node.gyp

```diff
@@ -310,11 +310,9 @@
       'target_name': '<(node_lib_target_name)',
       'type': '<(node_intermediate_lib_type)',
       'product_name': '<(node_core_target_name)',
-
       'dependencies': [
-        'node_js2c#host',
+        'mkssldef',
```

Comment:
Is there a reason this is made a dependency? It looks like an action in the same spirit of js2c

Reply:
Moved.